EIPS/eip-1.md

```diff
@@ -96,13 +98,13 @@ Each EIP should have the following parts:
 
 - Preamble - RFC 822 style headers containing metadata about the EIP, including the EIP number, a short descriptive title (limited to a maximum of 44 characters), and the author details. See [below](./eip-1.md#eip-header-preamble) for details.
 - Abstract - A short (~200 word) description of the technical issue being addressed.
-- Motivation (*optional) - The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.
+- Motivation (*optional) - A motivation section is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.
 - Specification - The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (cpp-ethereum, go-ethereum, parity, ethereumJ, ethereumjs-lib, [and others](https://github.com/ethereum/wiki/wiki/Clients).
-- Rationale - The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.
+- Rationale - The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussions.
```

Comment:
I think `discussion` is correct here.  While technically there may be multiple discussions, I believe `discussion` is one of those words that can be used in a sort-of-plurality sort of way.

Reply:
We use `discussions` in this section multiple times in the next few lines. So although both can work, I changed to `discussions` for consistency.

Reply:
There appears to be a bit of a mix `discussion` and `discussions` throughout this document.  I just went through them and feel like they are all used correctly currently.

Thinking and researching more on why I think this should be `discussion`, I believe it is because its usage here is as an uncountable noun, while `discussions` is the plural form of a countable noun.  `discussion` can be either a countable noun or an uncountable noun, and which one is appropriate depends on the context.  In this particular context, I am reasonably confident that the uncountable noun version is appropriate.

Reply:
Reverted to discussion and added a missing article